traja/tests/test_trajectory.py

```diff
@@ -221,17 +221,10 @@ def test_transition_matrix():
     transitions_matrix = traja.transition_matrix(grid_indices1D)
 
 
-def test_calc_laterality():
-    df_copy = df.copy()
-    right_turns, left_turns = traja.calc_laterality(df, dist_thresh=1)
-    assert left_turns == 4
-    assert right_turns == 0
-
-
-def test_calc_flow_angles():
+def test_calculate_flow_angles():
     df_copy = df.copy()
     grid_indices = traja.grid_coordinates(df_copy)
-    U, V = traja.calc_flow_angles(grid_indices.values)
+    U, V = traja.calculate_flow_angles(grid_indices.values)
```

Comment:
@aleczoeller your fork does not show the most recent changes, like this shortening to “calc”. Please update your branch to reflect the latest master branch

Reply:
Hi Justin, this has been updated.  It looked like there was a CI build failed test, but I couldn't see the specifics on the cause